modules/util/convert/convert_pixart_diffusers_to_ckpt.py

```diff
@@ -1,44 +1,63 @@
+import modules.util.convert.convert_diffusers_to_ckpt_util as util
+from modules.util.enum.ModelType import ModelType
+
 import torch
-from diffusers.models.embeddings import PatchEmbed
 from torch import Tensor
 
-import modules.util.convert.convert_diffusers_to_ckpt_util as util
-from modules.util.enum.ModelType import ModelType
+from diffusers.models.embeddings import PatchEmbed
 
 
 def __map_transformer_attention_block(in_states: dict, out_prefix: str, in_prefix: str) -> dict:
     out_states = {}
 
-    attn_qkv_bias = torch.cat([
-        in_states[util.combine(in_prefix, "attn1.to_q.bias")],
-        in_states[util.combine(in_prefix, "attn1.to_k.bias")],
-        in_states[util.combine(in_prefix, "attn1.to_v.bias")],
-    ], 0)
-
-    attn_qkv_weight = torch.cat([
-        in_states[util.combine(in_prefix, "attn1.to_q.weight")],
-        in_states[util.combine(in_prefix, "attn1.to_k.weight")],
-        in_states[util.combine(in_prefix, "attn1.to_v.weight")],
-    ], 0)
-
-    cross_attn_kv_linear_bias = torch.cat([
-        in_states[util.combine(in_prefix, "attn2.to_k.bias")],
-        in_states[util.combine(in_prefix, "attn2.to_v.bias")],
-    ], 0)
-
-    cross_attn_kv_linear_weight = torch.cat([
-        in_states[util.combine(in_prefix, "attn2.to_k.weight")],
-        in_states[util.combine(in_prefix, "attn2.to_v.weight")],
-    ], 0)
+    attn_qkv_bias = torch.cat(
```

Comment:
These convert functions (this file and the ones in the same directory) should also be handled differently. If possible, at least the line length should be increased. At the moment, each parameter mapping is done in one line. adding line breaks in random places just because the line was a little over 120 characters makes the code less readable

Reply:
Disabled formatter.